lenovo/legion/16ach6h/default.nix

```diff
@@ -7,6 +7,12 @@
     # This specialisation is for the case where "DDG" (Dual-Direct GFX, A hardware feature that can enable in bios) is enabled, since the amd igpu is blocked at hardware level and the built-in display is directly connected to the dgpu, we no longer need the amdgpu and prime configuration.
     system.nixos.tags = [ "Dual-Direct-GFX-Mode" ];
     services.xserver.videoDrivers = [ "nvidia" ]; # This will override services.xserver.videoDrivers = lib.mkDefault [ "amdgpu" "nvidia" ];
-    hardware.nvidia.prime.offload.enable = false;
+    hardware = {
+      nvidia.prime.offload.enable = false;
+      amdgpu = {
+        amdvlk = false;
+        opencl = false;
```

Comment:
Can you write a simple comment similar to what you described in your PR also here and the other case below where you disable the options?